src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/models/accrual/Accrual.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.enums.AccrualType;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
```

Comment:
We have code smells regarding unused private fields. I think we can rectify this by limiting the getter and setter to specific fields.

Currently we don't access id, tenantId, personId & Contributions

Reply:
These are false positives as highlighted in this thread: https://community.sonarsource.com/t/false-positive-with-lombok-and-data-unused-private-fields-should-be-removed/83955/1
This issue it was resolved on SonarQube but not on SonarCloud from the looks of it.
I've asked a question on Sonar Community: https://community.sonarsource.com/t/false-positive-in-sonarcloud-with-lombok-annotations-unused-private-fields-should-be-removed/90355

In the meantime I've marked the code smells as false positives